less/reset.less

```diff
@@ -45,4 +45,40 @@ q:before, q:after {
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
-}
\ No newline at end of file
+}
+
+//QAs
+/* 
+
+1. What is the difference between an adaptive website and a fully responsive website?
+Adaptive website is necessarily set on a fluid grid; targets specific device resolutions.
+Responsive website is built on a fluid grid; will change with the browser, no matter what the screen resolution and content can break apart and realign.
+Fluid(%) + Adaptive = Responsive
+
+
+2. Describe what it means to be mobile first vs desktop first.
+
+Mobile First:
+Base styles are for mobile
+Media queries use min-width
+Media queries go from lowwer to higher numbers
+
+Desktop First:
+Base styles are for desktop
+Media queries use max-width
+Media queries go from higher to lower numbers
+
+3. What does font-size: 62.5% in the html tag do for us when using rem units?
+It is a trick recommended by Josh. It can be used on a webpage if you use rem units.
```

Comment:
Do you understand why 62.5% is used? Its deals with the conversion from rem to px. If we set the html tag to 100%, we would have to do the conversions ourselves which is no fun. With 62.5%, we know that 1.2rem === 12 px.

Reply:
Now, it makes more sense to me. Thanks for elaborating it!